stdlib/public/runtime/EnumImpl.h

```diff
@@ -170,14 +170,14 @@ inline void storeEnumTagSinglePayloadImpl(
 
     // Store into the value.
 #if defined(__BIG_ENDIAN__)
```

Comment:
This really needs to be key'ed off of the triple.  If you are fixing the big endian codepath, please fix that.

Reply:
Isn't the triple for codegen rather than the stdlib/runtime (which is always compiled for the target architecture)?

If we do want to change how we select which code to use we should do that in a separate fix in my opinion.

Reply:
This code has remained this way for some time now, and I think that it has come up a couple of times, that IRGen needs to be able to test this on little endian and big endian.

Reply:
I'm sorry, I still don't understand what you want to change in this PR?

We should definitely change IRGen to use triples, but that is a completely separate issue from this PR AFAIK.

Reply:
Yes, I agree that it is a different issue.  However, since you are already changing the code around it, I would like that you take care of that, especially since this is not intended for the 5.0 branch.

Reply:
OK I've added another commit to make those changes to IRGen.